wiki/Help_Centre/en.md

```diff
@@ -929,36 +987,47 @@ These files are extremely handy to have and can make solving even very complicat
 
-There's a few things we can try to resolve this issue.
+**Event Viewer is a built in component of Windows which can be used to find crash logs when osu! does not provide you with any. These crash logs can be used to help determine what is causing your issue.**
 
-##### Have you entered your password correctly?
+When having problems with osu! crashing, if osu! doesn't give you a crash log the only place to find one is through Event Viewer.
 
-**This is the most obvious one.**
+If a support team member has asked you to find a crash log in Event Viewer, here's how to do it:
 
-If you've forgotten what your password is, you can use the [password recovery page](https://osu.ppy.sh/home/password-reset) to recover access to your account.
+1. After osu! has crashed, press `Win` + `R`.
+2. In the run box type `eventvwr` and press Enter. This will open the Event Viewer.
+3. In Event Viewer, on the left, go to `Windows Logs > Application`.
+4. On the right click `Filter current log`.
+5. 6. On the filter window that opens, make sure you have the Error box checked and press OK.
+6. Press `Ctrl` + `F` and type osu! in the find box. It will find the first crash log from osu!.
+7. Go in to the `Details` tab, expand System and Event Data by clicking on each of them.
+8. Copy the text from there and paste it in to your forum post.
 
-##### Are you using any proxy servers or VPNs?
+### Frame Time Graph
 
-**Disabling these before attempting to connect again will likely fix your issue.**
+#### What is the 'Frame Time Graph' and how do I use it?
 
-Some VPN providers and proxy servers simply do not work with osu! for whatever reason.
+**The Frame Time Graph is a feature of osu! that can help us gather more useful information on performance issues you may be experiencing**
 
-##### I'm still getting the error!
+When faced with tricky perofmance issues in osu!, the Frame Time Graph is a good tool to help us diagnose the issue at hand, and help you find a solution to the issue.
 
-**We'll need a `network.log` file from you to figure out what's going on if neither of the above solutions work.**
+If a support team member has asked you to take a video or screenshot with the Frame Time Graph open, here's how to do it:
 
-Check out the article a section above on how to get a log file to send to us.
+1. Open osu!.
+2. Press `Ctrl` + `F11`. to open the Frame Time Graph.
+3. Trigger or wait for the performance issue to occur.
+4. Take a screenshot by pressing `Shift` + `F12`. This will open a browser tab with the screenshot in it, copy and paste the link provided into your forum post.
+5. Close the Frame Time Graph the same way you opened it, by pressing `Ctrl` + `F11`.
 
-Rarely, you might need to contact your ISP to inquire about these problems. Certain ISPs have historically had issues connecting to osu!, usually due to poor routing.
+If you need to record a video of this, you will need to download a video recording software and use that.
```

Comment:
Is this line necessary?

Reply:
Probably not honestly. added it because I mentioned video above. I think I'll remove this.